### DISCUSSION TITLE
BT-1451 Winner Decision Date

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#678] [Q&A] [answered]
Hello Labs,

BT-1451 Winner Decision Date
I see if there is a winner, we have to fill this field with the decision date.

But if there is no winner ... don't we have to fill this field (with decision date)?

Thanks in advance
Anna

**Comment by YvesJo** [comment, marked as answer]
Hi,
The date the Buyer made his decision about the winning tender has been identified in the Regulation Annex as a property of the Contract (BG-310), therefore in absence of any contract, which is the case in the absence of a winner, it is not possible to report this information.

**Reply by AnnaKSZ** [reply]
Thanks for the quick response
